hibernate-core/src/main/java/org/hibernate/tool/hbm2ddl/SchemaExport.java

```diff
@@ -159,6 +160,18 @@ public SchemaExport setOutputFile(String filename) {
 		return this;
 	}
 
+	/**
+	 * For generating a export script file, by default the content will be appended at the begin or end of the file.
+	 *
+	 * the sql will be written to the end of the file rather than the beginning
+	 *
+	 * @return this
+	 */
+	public SchemaExport setOverrideOutputFileContent(){
```

Comment:
I hate to focus on such minor details, but adding space delimiter before '{' is definitively better

Reply:
Thanks @NathanQingyangXu , fixed it!